inline-images is not BC because gulp-image-data-uri is not installed

The inline-images task breaks projects that don't have the gulp-image-data-uri installed.

This can most likely be fixed when using the plugin versions of the plugins instead of defining them in the task.

Current error when running gulp without gulp-image-data-uri installed

```
module.js:338
    throw err;
          ^
Error: Cannot find module 'gulp-image-data-uri'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at module.exports (/srv/www/sources/sigravitaliteitshuis/var/frontend/bower_components/gulp-tasks/tasks/inline-images.js:7:24)
    at createTask (/srv/www/sources/sigravitaliteitshuis/var/frontend/node_modules/gulp-simple-task-loader/build/processor.js:57:43)
    at handleFileOrDirectory (/srv/www/sources/sigravitaliteitshuis/var/frontend/node_modules/gulp-simple-task-loader/build/processor.js:52:13)
    at Array.forEach (native)
    at processDirectory (/srv/www/sources/sigravitaliteitshuis/var/frontend/node_modules/gulp-simple-task-loader/build/processor.js:73:67)
    at Object.processTaskDirectory (/srv/www/sources/sigravitaliteitshuis/var/frontend/node_modules/gulp-simple-task-loader/build/processor.js:76:7)
```

--- a/tasks/inline-images.js
+++ b/tasks/inline-images.js
@@ -4,23 +4,20 @@
 'use strict';
 
 module.exports = function(gulp, config, plugins) {
-    var imageDataURI = require('gulp-image-data-uri');
-    var concat = require('gulp-concat');
-    var addsrc = require('gulp-add-src');
 
     return function() {
         return gulp.src(config['inline-images'].src)
             // Create the data uris of the images
-            .pipe(imageDataURI({
+            .pipe(plugins.imageDataUri({
                 template: {
                     file: __dirname + '/inline-images/image-data-uri-template.scss'
                 }
             }))
             // Prepend and append scss to make the mixins
-            .pipe(addsrc.prepend(__dirname + '/inline-images/_inline-images-prepend.scss'))
-            .pipe(addsrc.append(__dirname + '/inline-images/_inline-images-append.scss'))
+            .pipe(plugins.addSrc.prepend(__dirname + '/inline-images/_inline-images-prepend.scss'))
+            .pipe(plugins.addSrc.append(__dirname + '/inline-images/_inline-images-append.scss'))
             // Output to generated scss file
-            .pipe(concat('_inline-images.scss'))
+            .pipe(plugins.concat('_inline-images.scss'))
             .pipe(gulp.dest('./style/generated'));
     };
 };

